src/components/SecondaryPanes/Breakpoints/index.js

```diff
@@ -81,41 +81,45 @@ class Breakpoints extends Component<Props> {
       ...breakpointSources.map(({ source, breakpoints }) => source)
     ];
 
-    return [
-      ...breakpointSources.map(({ source, breakpoints, i }) => {
-        const path = getDisplayPath(source, sources);
-        const sortedBreakpoints = sortSelectedBreakpoints(
-          breakpoints,
-          selectedSource
-        );
-
-        return [
-          <BreakpointHeading
-            source={source}
-            sources={sources}
-            path={path}
-            key={source.url}
-          />,
-          ...sortedBreakpoints.map(breakpoint => (
-            <Breakpoint
-              breakpoint={breakpoint}
-              source={source}
-              selectedSource={selectedSource}
-              key={makeBreakpointId(
-                getSelectedLocation(breakpoint, selectedSource)
-              )}
-            />
-          ))
-        ];
-      })
-    ];
+    return (
+      <div className="pane breakpoints-list">
+        {[
+          ...breakpointSources.map(({ source, breakpoints, i }) => {
+            const path = getDisplayPath(source, sources);
+            const sortedBreakpoints = sortSelectedBreakpoints(
+              breakpoints,
+              selectedSource
+            );
+
+            return [
+              <BreakpointHeading
+                source={source}
+                sources={sources}
+                path={path}
+                key={source.url}
+              />,
+              ...sortedBreakpoints.map(breakpoint => (
+                <Breakpoint
+                  breakpoint={breakpoint}
+                  source={source}
+                  selectedSource={selectedSource}
+                  key={makeBreakpointId(
+                    getSelectedLocation(breakpoint, selectedSource)
+                  )}
+                />
+              ))
+            ];
+          })
+        ]}
+      </div>
+    );
   }
 
   render() {
     return (
-      <div className="pane breakpoints-list">
+      <div>
```

Comment:
It might be useful to still have a specific class that identifies the content of the "Breakpoints" pane.
It's likely we have some mochitests that select DOM nodes using this class, so we could be breaking that.

Maybe we should add a class to the element we're trying to style, rather than move `breakpoints-list` to a different element.

Reply:
mochitest selectors:
```
  breakpointItem: i => `.breakpoints-list div:nth-of-type(${i})`,
  breakpointItems: ".breakpoints-list .breakpoint",
```
But maybe you already accounted for that.

Reply:
Tests broke initially because of that, but I've fixed the relevant tests. The reason for structuring it this way was consistency with XHR Breakpoints:

```html
<!-- XHR Breakpoints -->
<div>
  <div class="breakpoints-exceptions-options"> ... </div>
  <ul class="pane expressions-list"> ... </ul>
</div>

<!-- Breakpoints -->
<div>
  <div class="breakpoints-exceptions"> ... </div>
  <div class="pane breakpoints-list"> ... </div>
</div>
```